webapp/sass/layout/_markdown.scss

```diff
@@ -3,6 +3,13 @@
 .markdown__heading {
     font-weight: 700;
     line-height: 1.5;
+
+    .emoticon {
```

Comment:
I mean that the rules you had here before worked fine when just added to the root `.emoticon` class instead of to `.markdown__heading .emoticon`. That way, everything with `.emoticon`  would work the same and we'd be less likely to conflicts between the rules in those two places